react/ContactsListModal/AddContactButton.jsx

```diff
@@ -1,22 +1,50 @@
-import React from 'react'
-import { withClient } from 'cozy-client'
+import React, { useEffect } from 'react'
+import { withClient, models, queryConnect } from 'cozy-client'
+import CozyRealtime from 'cozy-realtime'
 import AppLinker, { generateWebLink } from '../AppLinker'
 import { ButtonLink } from '../Button'
+import compose from 'lodash/flowRight'
 
 const DumbAddContactButton = props => {
-  const { client, ...rest } = props
-
-  const cozyURL = new URL(client.getStackClient().uri)
-  const { cozySubdomainType } = client.getInstanceOptions()
-  const contactsAppSlug = 'contacts'
-  const contactsAppHref = generateWebLink({
-    cozyUrl: cozyURL.origin,
-    slug: contactsAppSlug,
-    subDomainType: cozySubdomainType
+  const { client, apps, ...rest } = props
+
+  const wantedApp = { slug: 'contacts' }
+  const installedApp = models.applications.isInstalled(apps.data, {
+    slug: 'contacts'
   })
 
+  let href
+
+  if (installedApp) {
+    const cozyURL = new URL(client.getStackClient().uri)
+    const { cozySubdomainType } = client.getInstanceOptions()
+    href = generateWebLink({
+      cozyUrl: cozyURL.origin,
+      slug: installedApp.attributes.slug,
+      subDomainType: cozySubdomainType
+    })
```

Comment:
Is not the responsability of this PR but https://github.com/cozy/cozy-ui/issues/1327

Reply:
Since you have the `installedApp` you can simply use `models.applications.getUrl(installedApp)` since we'll have the `app.links.related` from the stack ;)

Reply:
:+1: thankss